src/vsphere_cpi/data/swagger-nsxt-manager-template/api_client.mustache

```diff
@@ -89,6 +88,12 @@ module {{moduleName}}
       response
     end
 
+    # Enables the deletion of test artifacts such as NS Groups and Logical Routers
+    # Don't enable in production code; it enables Management API to modify Policy objects
+    def x_allow_overwrite(value = true)
+      @default_headers['X-Allow-Overwrite'] = value
+    end
```

Comment:
If I understand correctly, we don't have direct control over the `config` object passed to the ApiClient constructor, and thus need to call this separate method to update the headers. Is that correct?

Reply:
I think we could similarly override the Swagger generated configuration object (like we are elsewhere) or otherwise add an extra param to the NSXT API client (which is swagger generated, but which we already template). 

Initially it seemed like this functionality was more of a rush, and so I chose not to do either of these things. 

Long-term I'm not excited about having a lot of mutations in the swagger-generated parts with the anticipation that this is an additional maintenance burden. I think trade-off makes both changes equally unappealing and this one seemed a little faster (and also kind of preserves a separation between "our" config and the "NSXT API Client" config, which is already a little confusing). 

I dunno, we could make this more "seamless" but I can't yet come up with any really good reasons to do so.